fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/ContainmentIndex.java

```diff
@@ -124,4 +124,11 @@
      */
     void reset();
 
+    /**
+     * Find whether there are any resources that starts with the ID provided.
+     * @param txID The transaction id, or null if no transaction.
+     * @param fedoraId The ID to use to look for other IDs.
+     * @return Are there any matching IDs.
+     */
+    boolean hasResourcesStartingWith(final String txID, final FedoraId fedoraId);
```

Comment:
Maybe as a separate clean-up task, this interface should normalize on using `ID` or `Id`. Currently it is a complete mix.

Reply:
I'll clean it up now.